Frontend/src/Components/Motor_Motion/Motor_Motion.js

```diff
@@ -0,0 +1,448 @@
+import {Grid, GridItem, Box, Flex, Spacer, Text, Center, useColorMode, VStack, Image, SimpleGrid} from "@chakra-ui/react";
+import HeadingCell from "../Shared/HeadingCell";
+import CONSTANTS from "../../data-constants.json"
+import getColor from "../Shared/colors";
+import images from "../Motor_Motion/Icons/Images"
+import RangeCell from "../Shared/RangeCell";
+
+export default function Motor_Motion(props) {
+    const { colorMode } = useColorMode();
+
+    const borderCol = getColor("border", colorMode);
+    const greenBgCol = getColor("greenBg", colorMode);
+    const greenTxtCol = getColor("txtGreenBg", colorMode);
+    const redBgCol = getColor("redBg", colorMode);
+    const redTxtCol = getColor("txtRedBg", colorMode);
+    const headerBg = getColor("header", colorMode);
+
+    const fitType = "scale-down";
+
+    const Images = images[`${colorMode}`];
+    
+    return (
+        <Flex flex='auto' direction='column'>
+            <HeadingCell fontSize='2.2vh' label='Motor / Motion'/>
+            <Grid 
+                flex='2'
+                templateRows="1fr 1fr 1fr 1.4fr 1.4fr" 
+                templateColumns="1fr 1fr 4fr 2fr"
+                h='25%'
+                pt='1'
+                pb='1'
+            >
```

Comment:
```suggestion
            <Grid 
                flex='2'
                templateRows="1fr 1fr 1fr 1.4fr 1.4fr" 
                templateColumns="1fr 1fr 4fr 2fr"
                pt='1'
                pb='1'
            >
```
The `h` should be unnecessary